src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothSocket.cs

```diff
@@ -232,6 +233,59 @@ void IBluetoothClientSocket.Disconnect()
             }
         }
 
+        /// <summary>
+        /// Registers a callback function that will be invoked when the SCO(Synchronous Connection Oriented link) state is changed
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<SocketConnectionRequestedEventArgs> ConnectionRequested
+        {
+            add
+            {
+                if (_connectionRequested == null)
+                {
+                    RegisterConnectionRequestedChangedEvent();
+                }
+                _connectionRequested += value;
+            }
+            remove
+            {
+                _connectionRequested -= value;
+                if (_connectionRequested == null)
+                {
+                    UnregisterConnectionRequestedChangedEvent();
+                }
+            }
+        }
+
+        private void RegisterConnectionRequestedChangedEvent()
+        {
+            Interop.Bluetooth.SocketConnectionRequestedCallback _connectionRequestedCallback = (int socket_fd, string remoteAddress, IntPtr userData) =>
+            {
+                Log.Info(Globals.LogTag, "ConnectionStateChangedCallback is called");
+                if (_connectionRequested != null)
+                {
+                    _connectionRequested(null, new SocketConnectionRequestedEventArgs(socket_fd, remoteAddress)); // Ã¹¹øÂ° ÆÄ¶ó¹ÌÅÍ È®ÀÎ
+                }
+            };
+            
+            int ret = Interop.Bluetooth.SetSocketConnectionRequestedCallback(_connectionRequestedCallback, IntPtr.Zero);
```

Comment:
It seem to a global function on Core API (there is no context to handle a target, and only one callback can be set).
It can be issue when a BluetoothSocket was created more than one.

Reply:
Done. Thank  you.